src/Utopia/Messaging/Response.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Utopia\Messaging;
+
+class Response
+{
+    private int $deliveredTo;
+
+    private string $type;
+
+    /**
+     * @var array<string, string>
```

Comment:
Should this be `array<array<string, string>>` instead?

Reply:
yep, fixed this